tests/data/tokenizers/pretrained_transformer_tokenizer_test.py

```diff
@@ -13,7 +13,7 @@ def test_splits_roberta(self):
         sentence = "A, <mask> AllenNLP sentence."
         expected_tokens = [
             "<s>",
-            "ĠA",
+            "A",
```

Comment:
See this discussion: huggingface/transformers#5507

Reply:
Man. I hope that's the last we see of this stupid leading-space debate. Just last week @matt-peters and I talked about it again because we weren't sure which one is correct.

Reply:
🤦